src/WebSocketStreamQueue.ts

```diff
@@ -0,0 +1,89 @@
+// WebSocketStreamQueue can have 3 states regarding the head and the tail
+// if (head == null && head === tail) then the queue is empty
+// if (head != null && head === tail) then the queue has 1 item
+// if (head != null && head !== tail) then the queue has 2 or more items
+class WebSocketStreamQueue {
```

Comment:
If the comment is relevant to the class it should be docblock format.

Reply:
Was this done?

Reply:
yes